### PR TITLE
[IMP] Support branches in the environment variable VERSION of .travis.yml

### PR DESCRIPTION
**Pull request by JesusZapata**
Support branches in the environment variable VERSION of .travis.yml

Now
```
VERSION=master
ODOO_REPO=odoo/odoo
```
Expected
```
VERSION=pull/3
ODOO_REPO=odoo/odoo
```

In the enviroment variable `VERSION` can be used:
- [x] Branch : VERSION="master"
- [x] PR : VERSION="pull/10

Referring to issue #173

### CODE REVIEW

**Comment by moylop260** [comment]
Could you create a dummy PR from other project in order to test this one?

File: travis/test_server.py [hunk 1]
```diff
@@ -142,8 +143,11 @@ def get_addons_path(travis_home, travis_build_dir, server_path):
     return addons_path
 
 
-def get_server_script(odoo_version):
-    return 'odoo-bin' if float(odoo_version) >= 10 else 'openerp-server'
+def get_server_script(server_path):
+    if os.path.isfile(os.path.join(server_path, 'odoo-bin')):
```
Comment (moylop260) [inline travis/test_server.py]: I told you: Review the issue 3 times before of create a change...
You will see that the change was approved to fix it with `try-except` and not with `if isfile` 😞

Comment (JesusZapata) [inline travis/test_server.py]: Sorry! 
I know the try catch said but it is much easier!
Do not worry i'll fix it

**Comment by JesusZapata** [comment]
@moylop260 
I created https://github.com/Vauxoo/odoo/pull/167 and https://github.com/Vauxoo/enterprise-mx/pull/15

In https://github.com/Vauxoo/enterprise-mx/pull/15 the runbot say fails for other reason http://runbot.vauxoo.com/runbot/static/build/23132-15-cdd782/logs/job_20_test_all.txt

But the server ran!

**Comment by JesusZapata** [comment]
The tests I've done have worked
image

File: travis/test_server.py (hunk 1 above)
Comment (moylop260) [inline travis/test_server.py]: FYI we have a fix proposal from OCA project: https://github.com/OCA/maintainer-quality-tools/pull/409

Comment (moylop260) [inline travis/test_server.py]: Then, reverts the changes related with https://github.com/OCA/maintainer-quality-tools/issues/398 and we will merge it waiting the oca fix

Comment (JesusZapata) [inline travis/test_server.py]: Done!
According to comment use try-except https://github.com/OCA/maintainer-quality-tools/issues/398#issuecomment-267404414

**Comment by moylop260** [comment]
Could you add a section in the root README to got how to works this feature in other PR?
and send a email to our team

**Comment by JesusZapata** [comment]
Done! 
I created new pull request https://github.com/Vauxoo/maintainer-quality-tools/pull/175 for the documentation of this pull request

**Comment by JesusZapata** [comment]
@moylop260 
Should I create a new PR for OCA?

**Comment by moylop260** [comment]
Could you create a new PR for OCA (with doc too)?

**Comment by JesusZapata** [comment]
@moylop260 
I created the PR in OCA https://github.com/OCA/maintainer-quality-tools/pull/428 and another PR in OCA for the documentation https://github.com/OCA/maintainer-quality-tools/pull/429

**Pull request merged**